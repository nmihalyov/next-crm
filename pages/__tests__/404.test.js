import NotFoundPage from '../404';

describe('NotFoundPage:', () => {
  it('renders 404 page', () => {
    const component = shallow(<NotFoundPage />);

    expect(component).toMatchSnapshot();
  });
});