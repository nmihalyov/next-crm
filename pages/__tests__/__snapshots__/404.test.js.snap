// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage: renders 404 page 1`] = `
<MainLayout
  title="Page not found"
>
  <Result
    extra={
      <Button
        type="primary"
      >
        <Link
          href="/"
        >
          Go to main page
        </Link>
      </Button>
    }
    status="404"
    subTitle="Page not found"
    title="404"
  />
</MainLayout>
`;
